src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -50,30 +51,14 @@ public DispatchData getData() {
 
     @Deprecated
     public Boolean update(int expiryTime) {
-        return this.update((long) expiryTime);
+        return false;
```

Comment:
shouldn't this simply return "true" now as it will instantly have successfully updated?

Reply:
this is not longer part of backpage, is not deprecated and should be deleted